client/state/stats/lists/constants.js

```diff
@@ -0,0 +1,26 @@
+export const PUBLICIZE_SERVICES_LABEL_ICON = {
+	twitter: {
+		label: 'Twitter',
```

Comment:
Did not know about that endpoint.  I know at the time this component was created, the icons were also hard-coded into the core apps as well.  Using the endpoint in all places would be a win for consistency.